### DISCUSSION TITLE
SideBar form focus problem

### DISCUSSION THREAD

**Discussion by erdemsusam** [primefaces/community#107] [PrimeVue]
Hello,
Whenever Sidebar is opened, chrome prints this warning:
Autofocus processing was blocked because a document already has a focused element.

I have a form inside a Sidebar and I want to focus on the first InputText when Sidebar is open.

Thanks for advance.

**Comment by tugcekucukoglu** [comment]
Hello,

I've created an issue about autofocus functionality on the Sidebar and it's fixed for the next version. Due to accessibility, the close button gets an autofocus, and due to the different behavior of browsers, Chrome sends such information.

**Comment by robsongajunior** [comment]
Do we have any status update?
